src/any_agent/serving/a2a/agent_executor.py

```diff
@@ -60,34 +60,41 @@ def __init__(self, agent: "AnyAgent", context_manager: ContextManager):
         self.context_manager = context_manager
 
     @override
-    async def execute(  # type: ignore[misc]
+    async def execute(
         self,
         context: RequestContext,
         event_queue: EventQueue,
     ) -> None:
         query = context.get_user_input()
         task = context.current_task
 
-        context_id = context.message.contextId
-        if not self.context_manager.get_context(context_id):
-            self.context_manager.add_context(context_id)
+        # FIXME context.message may be None
+        context_id = context.message.contextId  # type: ignore[union-attr]
+        if not self.context_manager.get_context(context_id):  # type: ignore[arg-type]
+            self.context_manager.add_context(context_id)  # type: ignore[arg-type]
 
         # Extract or create task ID
         if not task:
-            task = new_task(context.message)
-            await event_queue.enqueue_event(task)
+            if context.message is not None:
+                task = new_task(context.message)
+                await event_queue.enqueue_event(task)
+            else:
+                msg = "Task does not exist but the message in context is None"
+                logger.warning(msg)
+                raise ValueError(msg)
```

Comment:
This is a behavior change. Why should this throw an exception?

Reply:
Because the alternative where `context.message` is None is not considered in the original code, please check https://github.com/a2aproject/a2a-python/blob/1022093110100da27f040be4b35831bf8b1fe094/src/a2a/utils/task.py#L8
By default, I'll use exceptions for these cases unless I can come up with a suitable alternative, which I couldn't think about in this case :-/ Maybe we can just leave everything as it is if the Message and the Task are both None, but imho getting an exception thrown is the surest way to notice possible corner cases.